<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZerocoinPage</class>
 <widget class="QWidget" name="ZerocoinPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>380</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Right-click to edit address or label</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="denomination">
       <property name="text">
        <string>Select denomination</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="zerocoinAmount"/>
     </item>
     <item>
      <widget class="QPushButton" name="zerocoinMintButton">
       <property name="toolTip">
        <string>Renew Smartcash</string>
       </property>
       <property name="text">
        <string>&amp;Renew Smartcash</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../res/icons/add.png</normaloff>../res/icons/add.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zerocoinSpendButton">
       <property name="toolTip">
        <string>Reclaim Smartcash</string>
       </property>
       <property name="text">
        <string>&amp;Reclaim Smartcash</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../res/icons/edit.png</normaloff>../res/icons/edit.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="toolTip">
        <string>Export the data in the current tab to a file</string>
       </property>
       <property name="text">
        <string>&amp;Export</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../res/icons/export.png</normaloff>../res/icons/export.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
